lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -1280,7 +1280,7 @@ class ExprEmitter : public EmitterBase,
   SubExprInfo visitComb(ShrSOp op) {
     // >>> is only an arithmetic shift right when both operands are signed.
     // Otherwise it does a logical shift.
-    return emitBinary(op, LowestPrecedence, ">>>",
+    return emitBinary(op, Shift, ">>>",
```

Comment:
nice.  Please adjust the enumerator in VerilogPrecedence to ` Shift,           // << , >>, <<<, >>>`

Reply:
No change needed :)